Python/Product/EnvironmentsList/EnvironmentPathsExtension.xaml

```diff
@@ -36,6 +36,7 @@
                                     HorizontalAlignment="Stretch"
                                     Command="{x:Static l:EnvironmentView.MakeGlobalDefault}"
```

Comment:
Does the command still fire if you have a click event?

Reply:
yes!

Reply:
I was worried by that as well.